### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rshym**
Describe issue regarding outdated PyGitHub
Describe usage of '--branch' parameter
Minor fix of misalignment of text

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -138,3 +138,12 @@ build-history
                     │   ├── build-versions.inc
                     │   └── metadata-revs
                     └── prod-devel.xml
+
+Known issues
+============
+* If you use 'prod_pulls' parameter you may see message
+  "get_pulls() got an unexpected keyword argument 'base'".
+  This means that you have outdated PyGitHub on you PC.
+  PyGitHub v.1.45 is confirmed to works as expected.
```
Comment (andr2000) [inline README]: Imo, this means that we either need to put a `requirements.txt` or create `setup.py` to handle this
@lorc ?

Comment (rshym) [reply README]: Latter, after submitting this patch, I found solution to remove dependency on PyGitHub. This solution is not yet submitted, and will be provided as separate pull request.
So I will drop this commit "Known issues" as not relevant.

Sorry that I didn't remove this commit earlier.

**Comment by rshym** [comment]
Remove commit "Describe issue regarding outdated PyGitHub".
Dependency on PyGitHub is going to be removed, so there is no reason to merge info that will be reverted soon.

Branch is re-based on latest master.

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Comment by rshym** [comment]
Reviewer is added.

**Pull request merged**